readme.md

```diff
@@ -8,18 +8,20 @@ The Cloud File Manager is a Javascript library that enables applications to save
 * Browser LocalStorage (used mostly for development/testing)
 
 ## Development Setup
+    npm install
+    npm run build
+    npm run serve
```

Comment:
I'd suggest changing this to `npm [run] start` for consistency with every other project on the planet.